### PR TITLE
Add qlog tracer

### PR DESCRIPTION
**Pull request by matoval**
Add qlog tracer to the quic config when QLOGDIR environment variable is provided

### CODE REVIEW

**Comment by AaronH88** [comment]
The code is looking good but I am very surprised at the amount of changes in `go.sum` 
Can you please double check that this amount is correct and qVis dependencies have not been added accidentally

**Comment by matoval** [comment]
> The code is looking good but I am very surprised at the amount of changes in `go.sum` Can you please double check that this amount is correct and qVis dependencies have not been added accidentally

I double checked the deps and there was one unneeded dep that was added but the rest is need for using qlog

File: pkg/netceptor/conn.go [hunk 1]
```diff
@@ -90,6 +92,7 @@ func (s *Netceptor) listen(ctx context.Context, service string, tlscfg *tls.Conf
 	s.Logger.Debug("%s added service %s to listener registry", s.nodeID, service)
 	s.listenerRegistry[service] = pc
 	cfg := &quic.Config{
+		Tracer:                  s.tracer,
```
Comment (AaronH88) [inline pkg/netceptor/conn.go:95]: What happens if this is nil?

Comment (matoval) [reply pkg/netceptor/conn.go:95]: Quic checks if tracer != nil and handles if nil. DefaultConnectionTracer returns nil if qlogDir is not defined: https://github.com/quic-go/quic-go/blob/master/qlog/qlog_dir.go#L41

**Review by AaronH88 — APPROVED** [review]
lgtm

**Pull request merged**